src/index.html

```diff
@@ -6,11 +6,54 @@
       name="viewport"
       content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
     >
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta
+      http-equiv="X-UA-Compatible" content="ie=edge">
```

Comment:
do not change formatting here, it was ok before
![image](https://user-images.githubusercontent.com/64437769/142203109-2f94f57b-853f-4e54-8642-602dba143435.png)